src/ragger/navigator/__init__.py

```diff
@@ -13,7 +13,8 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 """
-from .navigator import NavInsID, NavIns, Navigator
+from .instruction import NavInsID, NavIns
+from .navigator import Navigator
```

Comment:
This is a breaking change that might be mentioned somewhere in the changelog?

Reply:
This should not break anything, these variables should still be available on every paths they were available previously. These imports are still valid:
- `from ragger.navigator import NavInsID, NavIns, Navigator`
- `from ragger.navigator.navigator import NavInsID, NavIns, Navigator`

Reply:
Great then!